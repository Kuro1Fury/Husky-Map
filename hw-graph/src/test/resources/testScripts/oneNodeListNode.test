# Tests that a graph can be created, a node can be added, and listNode works

CreateGraph A
AddNode A n1
ListNodes A
