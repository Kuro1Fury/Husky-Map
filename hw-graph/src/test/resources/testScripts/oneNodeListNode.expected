# Tests that a graph can be created, a node can be added, and listNode works

created graph A
added node n1 to A
A contains: n1